### PR TITLE
Facilitate using NEXTSTEP without --enable-tuning

### PR DESCRIPTION
**Pull request by vkip**

### CODE REVIEW

**Comment by blattms** [comment]
jenkins build this opm-simulators=5215 please

**Comment by vkip** [comment]
jenkins build this opm-simulators=5215 please

**Comment by blattms** [comment]
jenkins build this opm-simulators=5215 please

File: opm/input/eclipse/Schedule/ScheduleState.cpp [hunk 1]
```diff
@@ -379,8 +379,8 @@ Tuning& ScheduleState::tuning() {
 }
 
 // Returns -1 if there is no active limit on next step (from TUNING or NEXT[STEP])
-double ScheduleState::max_next_tstep() const {
-    double tuning_value = this->m_tuning.TSINIT.has_value() ? this->m_tuning.TSINIT.value() : -1.0;
+double ScheduleState::max_next_tstep(const bool enableTUNING) const {
+    double tuning_value = (enableTUNING && this->m_tuning.TSINIT.has_value())  ? this->m_tuning.TSINIT.value() : -1.0;
     double next_value = this->next_tstep.has_value() ? this->next_tstep->value() : -1.0;
     return std::max(next_value, tuning_value);
```
Comment (blattms) [inline opm/input/eclipse/Schedule/ScheduleState.cpp:385]: @vkip I know this was not part of the PR, but maybe you have an opinion on this. Should this rather be `std:min` instead? 
Taking the maximum here at least explains why we sometimes did not see an effect of NEXTSTEP (if it was smaller than an initial TSINIT)

Comment (vkip) [reply opm/input/eclipse/Schedule/ScheduleState.cpp:385]: Good point, std::min makes more sense I think.

Comment (blattms) [reply opm/input/eclipse/Schedule/ScheduleState.cpp:385]: Done in https://github.com/OPM/opm-common/pull/3954
